combat-trainer.lic

```diff
@@ -1791,7 +1791,35 @@ class TrainerProcess
 
     @pet_count = @skill_map.include?('Locks') ? get_boxes(@pet_box_source).count : 0
     echo("  @pet_count: #{@pet_count}") if $debug_mode_ct
-  end
+	
+	@constellations = get_data('constellations').constellations
+	
+    @have_divination_bones = DRC.get_settings.have_divination_bones
+    
+	@divination_bones_container = DRC.get_settings.divination_bones_storage['container']
+    
+	@divination_bones_tied = DRC.get_settings.divination_bones_storage['tied']
+    
+	@have_telescope = DRC.get_settings.have_telescope
+    
+	@ap_source = DRC.get_settings.astral_plane_training['train_source']
+    
+	@ap_destination = DRC.get_settings.astral_plane_training['train_destination']
+   
+    @prediction_pool_target = DRC.get_settings.astrology_use_full_pools ? 10 : DRC.get_settings.astrology_pool_target
+    
+	@rtr_data = nil
+    
+	@astrology_prediction_skills_magic = DRC.get_settings.astrology_prediction_skills['magic']
+    @astrology_prediction_skills_lore = DRC.get_settings.astrology_prediction_skills['lore']
+    @astrology_prediction_skills_offense = DRC.get_settings.astrology_prediction_skills['offense']
+    @astrology_prediction_skills_defense = DRC.get_settings.astrology_prediction_skills['defense']
+    @astrology_prediction_skills_survival = DRC.get_settings.astrology_prediction_skills['survival']
+	Flags.add('astro_true', 'You feel you have sufficiently pondered your latest observation')
+    echo " Flags['rtr-expire'].nil? #{Flags['rtr-expire'].nil?}" if UserVars.astrology_debug
```

Comment:
You dont need these debug statements here. No one is going to run rtr in combat

Reply:
took this out